libs/ledger-live-common/src/families/elrond/js-synchronisation.ts

```diff
@@ -63,17 +82,15 @@ const getAccountShape: GetAccountShape = async (info, syncConfig) => {
   return {
     id: accountId,
     balance: balance.plus(delegationBalance),
-    spendableBalance: balance.gt(FEES_BALANCE)
-      ? balance.minus(FEES_BALANCE)
-      : new BigNumber(0),
+    spendableBalance: balance,
```

Comment:
Any reason for that change? Can't `spendableBalance` be negative?

Reply:
This is also related to FEES_BALANCE which is not kept anymore.